_posts/2025-03-11-release-2.9.0.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: Release 2.9.0 of MDAnalysis
+---
+
+We are happy to release version 2.9.0 of MDAnalysis!
+
+This is a minor release of the MDAnalysis library, which means that it
+contains enhancements, bug fixes, deprecations, and other
+backwards-compatible changes.
```

Comment:
Is this actually true for this one? Should we mention https://github.com/MDAnalysis/mdanalysis/issues/4899?

Reply:
I think it's fine, thanks.